lib/msal-core/test/UserAgentApplication.spec.ts

```diff
@@ -1008,7 +1009,7 @@ describe("UserAgentApplication.ts Class", function () {
             cacheStorage.setItem(JSON.stringify(accessTokenKey), JSON.stringify(accessTokenValue));
 
             msal.acquireTokenSilent(tokenRequest).then(function(response) {
-                expect(response.scopes).to.be.deep.eq(tokenRequest.scopes);
+                expect(response.scopes).to.be.deep.eq(ScopeSet.trimAndConvertToLowerCase(tokenRequest.scopes));
```

Comment:
It would be better to hard code the expected response, instead of calling the same implementation detail again.

Reply:
done

Reply:
Still seeing the same code?

Reply:
Changes got overwritten for some reason. Updated.